vhdbuilder/packer/init-variables.sh

```diff
@@ -67,6 +67,10 @@ if [ "$MODE" == "sigMode" ]; then
 		echo "Gallery ${SIG_GALLERY_NAME} exists in the resource group ${AZURE_RESOURCE_GROUP_NAME} location ${AZURE_LOCATION}"
 	fi
 
+	if [[ "$OS_SKU" == "CBLMariner" ]]; then
+		SIG_IMAGE_NAME=${OS_SKU}${OS_VERSION//./}Gen2
```

Comment:
can we make the name more specific so we know it's imported from vhd, not created by packer?

Reply:
also it's not good to update/repurpose the var as it's confusing, can we introduce a new var?

Reply:
SIG_IMAGE_NAME is still only used as the the destination for Packer builds,
I've moved the image import code into init-variables and its more clear now that the imported image name is IMPORTED_IMAGE_NAME.

I've also saved both SIG_IMAGE_NAME and IMPORTED_IMAGE_NAME into the packer settings json, so the packer jsons can use the values directly from there.

Reply:
Thanks!